spec/models/like_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  before :all do
+    @user = User.create(name: 'Tom')
+    @post = Post.create(author: @user, title: 'Title')
+  end
+
+  context '#create' do
+    it 'is valid with existing user and post' do
+      expect(Like.new(user: @user, post: @post)).to be_valid
+    end
+
+    it 'is not valid without post' do
+      expect(Like.new(user: @user)).to_not be_valid
+    end
+
+    it 'is not valid without user' do
+      expect(Like.new(post: @post)).to_not be_valid
+    end
+  end
+
+  context '#update_comments_counter' do
```

Comment:
- [ ] **[OPTIONAL]** I think this subject should be about updating likes counter, not updating comments counter.